public/js/components/NumberCard.vue

```diff
@@ -28,7 +28,7 @@ export default {
       const lastMessage = this.messages.pop();
       if (lastMessage) {
         const value = evaluatePath(this.tile.property, lastMessage);
-        this.number = parseFloat(value).toFixed(1);
+        this.number = value === null ? "" : parseFloat(value).toFixed(1);
```

Comment:
I don’t think that’s particularly helpful. With `NaN`, the user can assume that something’s wrong with the data. An empty string hides this. Now it looks more like a glitch in the UI. Sometimes there are values and sometimes there aren’t.

Reply:
Fair enough. Reverted.